docs/source/tutorials/scripts/data_generation.py

```diff
@@ -0,0 +1,128 @@
+import matplotlib.pyplot as plt
```

Comment:
You've created a file in this folder:
docs/source/tutorials/scripts/__init__.py

Do we really need that file?